packages/web/src/pages/app/contributions.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { NextPage } from 'next';
+import { Heading, SimpleGrid, Text, VStack } from '@chakra-ui/react';
+import { AppLayout } from '../../components/Layout';
+import { ContributionsBox } from '../../components/Contributions';
+
+export interface ContributionsList {
+  id: string;
+  nodeID: string;
+  type: string;
+  score: number;
+  contributedAt: Date;
+  description: string | null;
+  url: string;
+}
+
+const Contributions: NextPage = () => {
+  const [Contributions, setContributions] = React.useState<ContributionsList[]>([]);
```

Comment:
Variable names should be lowercase otherwise IDEs could confuse them as Classes.

```suggestion
  const [contributions, setContributions] = React.useState<Contribution[]>([]);
```